teamcity/build_type.go

```diff
@@ -228,7 +228,7 @@ func newBuildTypeService(base *sling.Sling, httpClient *http.Client) *BuildTypeS
 }
 
 // Create Creates a new build type under a project
-func (s *BuildTypeService) Create(projectID string, buildType *BuildType) (*BuildTypeReference, error) {
+func (s *BuildTypeService) Create(buildType *BuildType) (*BuildTypeReference, error) {
```

Comment:
Although I totally understand the unused field (my bad there) - this would be a public API breaking change 😥.
The project follows SemVer and thus this would cause a major bump, which I'd reckon they should be sparse.

Would you mind to keep this unused parameter and maybe open an issue to track - then at a convenient time with a major release (like a next version of TeamCity), this can be tackled in a more timely fashion.

Reply:
Fixed - and opened 84